language/rflx_ast.py

```diff
@@ -600,3 +600,15 @@ class Select(Expr):
 
     expression = Field(type=Expr)
     selector = Field(type=UnqualifiedID)
+
+
+class Choice(Expr):
+
+    selectors = Field(type=RFLXNode.list)
```

Comment:
I suppose there is no better way to support `ID` as well as `NumericLiteral` as selectors?

Reply:
Unfortunately not at the moment. `RFLXNode` is their closest common ancestor (ID > AbstractID > RFLXNode, NumericLiteral > Expr > RFLXNode). We could introduce a `Literal` which we derive both from (I'm not sure whether `NumericLiteral` actually has to be an `Expr`). I doubt it's worth the effort, though.